kernels/streamer_matmul/Makefile

```diff
@@ -0,0 +1,40 @@
+# Courtesy of Federico Ficarelli
+
+.DEFAULT_GOAL := all
+
+include ../../runtime/snax-streamer-gemm.rules
+include ../../runtime/Makefile.rules
+
+TESTS =
+TESTS += half_tiled_matmul.x
+TESTS += matmul.x
+
+CFLAGS += -std=gnu11
+CFLAGS += -Wall -Wextra
+
+# Override snax-opt rules to avoid linalg-to-library-call pass
+SNAXOPTACCFLAGS = -p dispatch-kernels,set-memory-space,set-memory-layout,realize-memref-casts,insert-sync-barrier,dispatch-regions,insert-accfg-op{accelerator=snax_gemm},convert-linalg-to-accfg,convert-accfg-to-csr,snax-copy-to-dma,memref-to-snax,snax-to-func,clear-memory-space
+
```

Comment:
I think this is still running on CPU, can you remove the `*accfg*` passes and insert `convert-linalg-to-library-call` pass?

Reply:
Turns out these passes are not used anyways :D